packages/graphql/src/translate/where/create-element-where-and-params.ts

```diff
@@ -190,15 +190,22 @@ function createElementWhereAndParams({
                         relationship,
                         relationshipVariable: `${collectedMap}.relationship`,
                         parameterPrefix: `${parameterPrefix}.${fieldName}`,
+                        listPredicates: [currentListPredicate, ...(listPredicates || [])],
```

Comment:
**question** I'm not sure what this variable (`listPredicates` ) is, or how it works. Woudl appreciate a quick explanation :sweat_smile:

Reply:
`listPredicates` will store all list predicates (SINGLE, ANY, NONE,..) while (recursively) translating the where clause of the query. It can then be used in `create-element-where-and-params` to decide which list predicate to use plus if we want to set the `expectMultipleValues` for `apoc.cypher.runFirstColumn` to `false`.
Maybe a better name would be suitable? I couldn't come up with anything better..

Reply:
mmm, yeah, I think we could find a better name. but tbh I cannot think of much better name.

Maybe, for now, a comment explaining a bit what this is could help